Functions.Templates/Templates/CosmosDBTrigger-CSharp/build.config/template.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function"
+  ],
+  "name": "CosmosDBTrigger",
+  "identity": "Azure.Function.CSharp.CosmosDBTrigger",
+  "groupIdentity": "Azure.Function.CosmosDBTrigger",
+  "shortName": "CosmosDB",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "CosmosDBTrigger",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "Connection": {
+      "description": "The connection string to the service that contains the collection to be monitored.",
```

Comment:
The name of the app setting containing your Cosmos DB connection string.
or 
The name of the app setting connection string to the service that contains the collection to be monitored.